conans/client/command.py

```diff
@@ -714,36 +671,32 @@ def export(self, *args):
                                   keep_source=args.keep_source)
 
     def remove(self, *args):
-        """Removes packages or binaries matching pattern from local cache or remote.
+        """Removes all packages or binaries matching pattern from local cache or remote.
```

Comment:
No, it doesnt remove all, it can remove just one build folder, or just one package from the cache